charts/beyla/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: beyla
-version: 1.1.0
+version: 1.2.0
```

Comment:
Since this looks more as a fix than as a feature, I'd set the version to `1.1.1`.

Reply:
done